style.css

```diff
@@ -88,6 +89,327 @@ a {
   padding: 5px;
 }
 
+.categories-list li {
+  font-weight: 400;
+  font-size: 1.2rem;
+  line-height: 3rem;
+  color: #fff;
+}
+
 .social-media ul li {
   padding: 5px;
 }
+
+#recentWorks {
+  display: grid;
+  height: auto;
+  grid-template-columns: 100%;
+}
+
+#recentWorks h2 {
+  color: #1c1a19;
+  font-weight: 400;
+  font-size: 36px;
+  border-bottom: 1px solid #212121;
+  align-items: center;
+  margin-top: 2em;
+  padding: 1em;
+}
+
+.imgHeader {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 2vh;
+}
+
+.imgHeader img {
+  width: 340px;
+}
+
+#seeProjects {
+  position: relative;
+  height: 100%;
+}
+
+#works h3 {
+  color: #091e42;
+  font-size: 29px;
+  font-weight: 400;
+  font-family: 'Crete Round', sans-serif;
+  margin: 0;
+}
+
+.introInfo {
+  color: #091e42;
+  font-size: 15px;
+}
+
+#works {
+  margin: 0 11vw;
+}
+
+#works ul {
+  display: flex;
+  padding-left: 0;
+}
+
+#works ul li {
+  border: 1px solid #8993a4;
+  background: #fff;
+  padding: 10px 12px;
+  color: #091e42;
+  margin: 3px;
+  list-style-type: none;
+}
+
+#works button {
+  display: flex;
+  background: #ff6b00;
+  color: #fff;
+  align-items: center;
+  padding: 12px;
+  justify-content: center;
+  width: 110px;
+  margin-top: 3px;
+  border: none;
+  cursor: pointer;
+}
+
+#cardWork button {
+  width: 100%;
+  position: absolute;
+  background-color: #ff6b00;
+  padding: 15px 0;
+  text-align: center;
+  margin-top: 2.5vh;
+  font-family: 'Inter', sans-serif;
+  color: #fff;
+  border: none;
+  bottom: 0;
+  cursor: pointer;
+}
+
+#contact-me button {
+  width: 40%;
+  background-color: #ff6b00;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  color: #fff;
+  border: none;
+  cursor: pointer;
+  margin-top: 15px;
+  margin-bottom: 124px;
+  font-weight: 700;
+  font-size: 17px;
+  height: 48px;
+}
```

Comment:
Well done :+1: on implementing button click.
- [ ] Kindly implement the `button interactions`,  such as: `hover`,  `enable`, etc. as per the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md).

![Screenshot from 2022-12-06 19-29-59](https://user-images.githubusercontent.com/60083437/205925784-df99ce62-23a7-4649-b0e3-c2b2fba489de.png)

- Here is a code suggestion for your button interaction design given below.

![Screenshot from 2022-12-07 00-52-35](https://user-images.githubusercontent.com/60083437/206000314-03a0691a-1142-4741-b277-a9c2a04ea769.png)


```
/* Enabled */
color: #FFFFFF;
background: #F55800;

/* Hover */
color: F55800;
background: #FFF5E1;
border: 1px solid #F55800;

/* Pressed */
color: #F4F1F0;
background: #E05304;

/* Disabled */
color: #5E6C84;
background: #F4F1F0;
```